nifi-nar-bundles/nifi-mqtt-bundle/nifi-mqtt-processors/src/main/java/org/apache/nifi/processors/mqtt/ConsumeMQTT.java

```diff
@@ -144,8 +144,8 @@
             .build();
 
     public static final PropertyDescriptor PROP_QOS = new PropertyDescriptor.Builder()
-            .name("Quality of Service(QoS)")
-            .description("The Quality of Service(QoS) to receive the message with. Accepts values '0', '1' or '2'; '0' for 'at most once', '1' for 'at least once', '2' for 'exactly once'.")
+            .name("Quality of Service (QoS)")
```

Comment:
Unfortunately we can't change this as it would be a breaking change. You should add ``.displayName()`` in addition to ``.name``.

Reply:
That's right, sorry. Thank you.